APIs_Platform/AT_CmdParser/README.md

```diff
@@ -0,0 +1,36 @@
+## ATCmdParser Example ##
+
+Example application showing usage of the `ATCmdParser` class with the ESP8266 WiFi module.
+
+The program retrieves the FW version of the ESP8266 module connected over UART using AT commands. 
+
+##  Getting started
+
+1. Import the example
+1. Compile and generate binary   
+1. Open a serial console session with the target platform using the following parameters:
+   * **Baud rate:** 9600
+   * **Data bits:** 8
+   * **Stop bits:** 1
+   * **Parity:** None
+1. Flash the target with the compiled binary. 
+1. The serial console should display a similar output to below, indicating a successful AT communication:
```

Comment:
This is not an example so I was not expecting a "getting started" section. We tend to use a template in the Mbed OS examples and recommend using "mbed compile -m <TARGET> -t <TOOLCHAIN> --flash --sterm"